misk/src/main/kotlin/misk/web/metadata/DatabaseQueryMetadata.kt

```diff
@@ -40,6 +43,28 @@ data class DatabaseQueryMetadata(
       queryClass = ClassNameFormatter.format(queryClass::class),
       constraints = constraints,
       orders = orders,
-      selects = selects
+      selects = selects,
+      types = types
   )
+
+  data class ConstraintMetadata(
+    override val name: String,
+    override val parametersType: String,
```

Comment:
should we continue to use field for the parameter? 
the other options is to include a separate field for repeated constraints